src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -2348,6 +2361,13 @@
     <Field Name="WhenFalseLabel" Type="LabelSymbol" Null="disallow"/>
   </Node>
 
+  <Node Name="BoundLoweredIsPatternExpression" Base="BoundExpression" HasValidate="true">
```

Comment:
> <Node Name="BoundLoweredIsPatternExpression" Base="BoundExpression" HasValidate="true"> [](http://example.com/codeflow?start=1&length=88)

I am confused. This change looks like a change that has been merged in a different PR. Why does it show up in this PR as well?